src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier randomFigure = new FigureSupplier();
+        Figure[] figures = new Figure[10];
+        for (int i = 0; i < figures.length;i++) {
+            figures[i] = randomFigure.getRandomFigure();
```

Comment:
The first half of figures in this array should be generated with random parameters.